dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/resources/docker/basic/docker-compose.yaml

```diff
@@ -32,6 +32,27 @@ services:
       interval: 5s
       timeout: 60s
       retries: 120
+#    command:
+#      - /bin/bash
+#      - -c
+#      - 'sed -i "s|resource.storage.type=.*|resource.storage.type=hdfs|g" /opt/dolphinscheduler/standalone-server/conf/common.properties'
```

Comment:
Can we remove this comment?

Reply:
Sorry. I haven't finished this work yet. Currently only testing.

Reply:
A little tip, next time when your PR does not finish, you should mark it as a draft PR, https://github.blog/2019-02-14-introducing-draft-pull-requests/

Reply:
How can I mark this PR as as draft since it has been created?

Reply:
you could still mark formal PR to draft by click here.